### PR TITLE
Support RESOLWE_API_HOST environmental variable

### PR DESCRIPTION
**Pull request by kostko**

### CODE REVIEW

**Comment by dblenkus** [comment]
Also add options for setting username and password.

**Comment by mstajdohar** [comment]
Tests pass. I will merge and publish a new release.

**Comment by kostko** [comment]
Added username and password variables.

File: resdk/resolwe.py [hunk 1]
```diff
@@ -70,8 +70,18 @@ class Resolwe(object):
 
     """
 
-    def __init__(self, username=None, password=None, url=DEFAULT_URL):
+    def __init__(self, username=None, password=None, url=None):
         """Initialize attributes."""
+        if url is None:
+            # Try to get URL from environmental variable, otherwise fallback to default.
+            url = os.environ.get('RESOLWE_API_HOST', DEFAULT_URL)
```
Comment (mstajdohar) [inline resdk/resolwe.py]: Default could be written here directly instead in the `DEFAULT_URL` constant.

**Pull request merged**